alf/examples/actor_critic.py

```diff
@@ -153,49 +157,54 @@ def create_algorithm(env,
 @gin.configurable
 def create_environment(env_name='CartPole-v0',
                        env_load_fn=suite_gym.load,
-                       random_seed=0,
                        num_parallel_environments=30):
     """Create environment."""
     if num_parallel_environments == 1:
         py_env = env_load_fn(env_name)
     else:
+        if env_load_fn == suite_socialbot.load:
+            logging.info("suite_socialbot environment")
```

Comment:
config `suite_mario.load.wrap_with_process=0` in gin config file instead

Reply:
Since we always want wrap_with_process=0 in this situation, it's better to fix its behavior in code.